docs/_data/atlantic-2023-schedule.yaml

```diff
@@ -0,0 +1,112 @@
+writing_day:
+  - time: '13:00'
+    title: 'Platform opens'
+    duration: '01:00'
+  - duration: '01:00'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - duration: '00:30'
+    title: Team up and start working
+  - duration: '01:30'
+    title: Welcome Wagon conference introduction and platform tour
+  - duration: '01:30'
+    title: '🍲 Lunch Break - Determine length with table'
+  - duration: '03:00'
+    title: Welcome Wagon conference introduction and platform tour
+  - duration: '00:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '13:00'
+    title: Platform opens
+    duration: '00:15'
+  - duration: '00:45'
+    title: Welcome Wagon conference introduction and platform tour
+  - duration: '00:45'
+    title: Write the Docs Team - Community update
```

Comment:
For the last few conferences, this has been 20/25 minutes.

Reply:
:+1: